apps/interactiveevents/views.py

```diff
@@ -21,6 +21,7 @@ def get_context_data(self, **kwargs):
         context['extra_fields'] = \
             models.ExtraFieldsInteractiveEvent.objects.filter(
                 module=self.module).first()
+        context['phase_desc_limit'] = 75
         return context
```

Comment:
do we need this as well if we have the truncate?

Reply:
ah good catch. leftovers from before